py/sentry_relay/processing.py

```diff
@@ -166,6 +167,17 @@ def is_codeowners_path_match(value, pattern):
     )
 
 
+def validate_pii_selector_spec(selector):
```

Comment:
nit: Should we call this `validate_pii_selector`? As a user of this function I would not know what the `_spec` suffix refers to.

Reply:
Agreed with @jjbayer

Reply:
Renamed